src/main/java/MockClasses/MockMedicalService/MockAmbulance.java

```diff
@@ -6,9 +6,9 @@
 import java.util.Random;
 
 public class MockAmbulance {
-    private Random random = new Random();
     public MockAmbulance(String name){
         this.name = name;
+        Random random = new Random();
```

Comment:
bylo dobrze chodzilo zeby to bylo polem w klasie `private final Random random = new Random();`